src/app/EventManagement.cpp

```diff
@@ -33,6 +33,21 @@ namespace chip {
 namespace app {
 static EventManagement sInstance;
 
+#if CHIP_SYSTEM_CONFIG_NO_LOCKING
```

Comment:
I'm confused by why we're using this macro to conditionally enable/disable the use of scoped locks?

Reply:
Since this scope is originated from SystemMutex.cpp/h, I think it is for new product/os where system lock has not yet implemented